Scripts/NuGet/EdFi.Installer.SandboxAdmin/Install-EdFiOdsSandboxAdmin.psm1

```diff
@@ -232,10 +226,11 @@ function Invoke-TransformWebConfigAppSettings {
         }
 
         $Config.AppSettingsOverrides.GetEnumerator() | ForEach-Object {
-            $appSettings[$_.key] = $_.value.toString()
+            Merge-Hashtables $appSettings $_.value.toString()
         }
-
-        Set-ApplicationSettings -ConfigFile $Config.WebConfigLocation -AppSettings $appSettings
+       
+        $newDevelopmentSettingsPath = Join-Path $Config.WebConfigLocation "appsettings.json"
+        New-JsonFile $newDevelopmentSettingsPath $appSettings -Overwrite
```

Comment:
as discussed this needs to read the existing settings and merge the overrides.
something like this:
```powershell
$settings = Get-Content $settingsFile | ConvertFrom-Json | ConvertTo-Hashtable
$mergedSettings = Merge-Hashtables $settings, $Config.AppSettingsOverrides
New-JsonFile $settingsFile $mergedSettings -Overwrite
```

Reply:
Modified as per the comment